packages/cozy-device-helper/src/cordova.js

```diff
@@ -1,2 +1,3 @@
 // cordova
-export const isCordova = () => window.cordova !== undefined
+export const isCordova = () =>
```

Comment:
est ce qu'il ne faut pas utiliser `global` au lieu de `window`

```suggestion
export const isCordova = () => global.cordova !== undefined
```

Reply:
Non pourquoi ? cordova n'est attaché que sur `window`.